lang.json

```diff
@@ -104,7 +105,8 @@
     "maturity": "alpha",
     "exts": [
       ".cc",
-      ".cpp"
+      ".cpp",
+      ".h"
```

Comment:
Shouldn't this be `.hh` ?

Reply:
many C++ projects also use .h for their header.
But yes that means we will parse .h files twice, once for C, and once for C++.
But yes we could also add .hh